src/components/Pagination/Pagination.test.tsx

```diff
@@ -69,20 +69,18 @@ describe("Pagination with getPageHref", () => {
   });
 
   it("When pagination has 1 element, one element is shown with both buttons disabled", () => {
```

Comment:
This statement shoudl be changed to the following...
`When pagination has 1 element, pagination is not shown`